citi-onboarding/PubliEasy
# Feature/services

## What I did

- Created service component
- Created feature/services branch
- Created styled.ts and index.tsx from Services
- Changed theme from Services



## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout <branch name>`
   1. `git pull origin <branch name>`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

***
**Time spent:** 2h

**Resolves:** #<issue number>
